### PR TITLE
Create rescale_mode_percentile.py

### PR DESCRIPTION
**Pull request by NiklasTR**

### CODE REVIEW

File: rescale_mode_percentile.py [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+#
+##################################
+
+import cellprofiler.image
+import cellprofiler.module
+import cellprofiler.setting
+
+__doc__ = """\
+RescaleModePercentile
+=============
```
Comment (bethac07) [inline rescale_mode_percentile.py]: Can you expand this bar to the length of the line above it?

File: rescale_mode_percentile.py [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+**RescaleModePercentile** is an image processing module that rescales flourescent images.
```
Comment (bethac07) [inline rescale_mode_percentile.py]: fluorescent

File: rescale_mode_percentile.py [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+Supports 2D? Supports 3D? Respects masks?
+============ ============ ===============
+YES          NO           NO
+============ ============ ===============
+See also
+^^^^^^^^
+This module is related to the RescaleMeanSD module.
+What do I need as input?
+^^^^^^^^^^^^^^^^^^^^^^^^
+A .tiff flourescent image
```
Comment (bethac07) [inline rescale_mode_percentile.py]: fluorescent

File: rescale_mode_percentile.py (hunk 2 above)
Comment (NiklasTR) [reply rescale_mode_percentile.py]: Oops - things that happen if you are not a native speaker. Thank you

Comment (bethac07) [reply rescale_mode_percentile.py]: No problem! I look a the spelling of that word a lot more frequently than you probably do :)

**Pull request merged**